src/app.rs

```diff
@@ -232,6 +234,10 @@ impl App {
             output_components,
             syntax_mapping,
             pager: self.matches.value_of("pager"),
+            italics: match self.matches.value_of("enable-italics") {
```

Comment:
I like that you suggest `always` and `never`, but in this case, let's maybe use just `--italics=always/never`?

Reply:
Or maybe just
```
--enable-italics[=yes]  # to enable it, with or without "=yes"
--enable-italics=no     # to switch it off
```

what do you think?

Reply:
IMHO, to have consistency between flags we should keep `always` and `never`. Because other flags use the `auto, always and never` mostly.

Reply:
okay, then let's use `--italic-text=always|never`